tt1-blocks/functions.php

```diff
@@ -69,6 +69,8 @@ function tt1_blocks_setup() {
  */
 function tt1_blocks_scripts() {
```

Comment:
Since this function will enqueue the css files, it's name must be changed to reflect the actual situation. `tt1_blocks_scripts_and_styles()` seems like a good fit.

Reply:
I've updated this to `tt1_blocks_enqueue_styles` which I think is more descriptive (this function enqueues just stylesheets, no scripts). 

For what it's worth, the original `_blocks_scripts` function name was brought over from Twenty Twenty-One, and in that case, the function does enqueue both scripts and styles.